interface/admin.py

```diff
@@ -1,11 +1,17 @@
 from zipfile import ZipFile
```

Comment:
There is also a `submission.download()` part in [`SubmissionAdmin.download_archive()`](https://github.com/vmck/acs-interface/blob/master/interface/admin.py#L92) that could use a `try-except` block.

Reply:
Good catch, fixed, thanks!